code/modules/surgery/implant.dm

```diff
@@ -217,13 +217,14 @@
 			affected.implants -= obj
 
 			BITSET(target.hud_updateflag, IMPLOYAL_HUD)
-			obj.dropInto(target.loc)
+			obj.forceMove(target.loc)
```

Comment:
`dropInto()` is basically designed as a "safer" `forceMove()` where a destination can hold something if it wants to hold it, but you can see the default behavior here:
https://github.com/Yonaguni/EuropaStation/blob/496d020d534d4b5349aed174cf0e430dc307161e/code/game/atoms.dm#L491-L503

Reply:
I'm not sure why it's using `dropInto`, though. Thought it might be messing with psi atom updates (but it wasn't).